pytest_fixtures/core/sys.py

```diff
@@ -45,9 +45,9 @@ def proxy_port_range(default_sat):
 
 
 @pytest.fixture
-def install_cockpit_plugin(default_sat):
-    default_sat.register_to_dogfood()
-    default_sat.install_cockpit()
+def install_cockpit_plugin(destructive_sat):
```

Comment:
Nonblocking, renaming fixture, using `cockpit_sat` directly into test args, also CLI/UI can be called as `cockpit_sat.cli` and fixture call using `@pytest.mark.usefixtures` can be removed.
```suggestion
def cockpit_sat(destructive_sat):
```

Reply:
Updated.